server/auth/github.js

```diff
@@ -1,6 +1,6 @@
 const passport = require('passport');
 const router = require('express').Router();
-const GithubStrategy = require('passport-github2').Strategy;
+// const GithubStrategy = require('passport-github2').Strategy;
```

Comment:
The updates break for me with these lines commented out, but once they are commented back in, everything works! Great job on the login and signup page!